app/src/components/market/sections/market_create/steps/items/funding_and_fee_step.tsx

```diff
@@ -272,6 +279,8 @@ const FundingAndFeeStep: React.FC<Props> = (props: Props) => {
 
   const onCollateralChange = (token: Token | null) => {
     if (!token) return
+    const tokenAddressFormatted = token.address.toLowerCase()
+    setCurrentToken({ exists: allMarkets.includes(tokenAddressFormatted), token: token.symbol })
```

Comment:
Since `currentToken` is only set on collateral change, the warning message doesn't appear unless you actually select a currency from the dropdown. To fix this, I would `setCurrentToken` in a `useEffect` that only runs on component mount.